lib/services/log.js

```diff
@@ -1,45 +1,23 @@
 'use strict';
 
-var _ = require('lodash'),
-  chalk = require('chalk'),
-  path = require('path'),
-  winston = require('winston'),
-  util = require('util'),
-  logLevel = process.env.LOG || 'info';
+const clayLog = require('clay-log'),
+  pkg = require('../../package.json');
+var amphoraSearchLogInstance;
 
-// default to console logger, but pretty-print it
-winston.remove(winston.transports.Console);
-winston.add(winston.transports.Console, {
-  colorize: true
+// Initialize the logger
+clayLog.init({
+  name: 'amphora-search',
+  prettyPrint: true,
+  meta: {
+    amphoraSearchVersion: pkg.version
+  }
 });
-winston.default.transports.console.level = logLevel;
 
-/**
- * @param {*} obj
- * @returns {boolean}
- */
-function isError(obj) {
-  return _.isError(obj) || _.isObject(obj) && obj.stack && _.endsWith(obj.name, 'Error');
-}
-
-/**
- * @param {string} dirname
- * @returns {Function}
- */
-module.exports.withStandardPrefix = function (dirname) {
-  const prefix = path.relative(process.cwd(), dirname).replace(/\.js$/, '').replace(/^[\.]\.\//, '');
+// Store the instance
+amphoraSearchLogInstance = clayLog.getLogger();
```

Comment:
oh, I see. you're pulling in the logger here, then referencing this one in all the other files. I guess that makes sense, but it's kinda confusing to reason about.

Reply:
The problem is that the `clay-log` module can't (shouldn't) maintain a reference to all the individual loggers that are instantiated in one process. Since Amphora, Amphora Search, Amphora HTML _and_ a Clay instance could all use the module, each individual package needs to maintain a reference to its own specific log instance so that each package is only logging messages with the correctly associated metadata

Reply:
Agreed